crates/swc_ecma_minifier/tests/terser/compress/functions/issue_2531_2/output.js

```diff
@@ -1,5 +1,4 @@
 function outer() {
-    var value;
```

Comment:
I think the change to these tests should be considered carefully

Reply:
This one is fine, issue_1606 is wrong